module/HostSystemCollector.py

```diff
@@ -0,0 +1,62 @@
+import os
+from BaseCollector import BaseCollector
+from prometheus_client.core import GaugeMetricFamily
+from tools.get_metrics import get_metric
+
+
+class HostSystemCollector(BaseCollector):
+
+    def __init__(self, resources, user, password):
+        self._resources = resources
+        self._user = user
+        self._password = password
+        self._statkeys = [{"name": "hardware_number_of_cpu_cores_info",
+                         "statkey": "hardware|cpuinfo|numCpuCores)"},
+                        {"name": "hardware_number_of_cpu_sockets_info",
+                         "statkey": "cpu|numpackages"},
+                        {"name": "memory_swap_used_kilobytes",
+                         "statkey": "mem|swapused_average"},
+                        {"name": "memory_useable_kilobytes",
+                         "statkey": "mem|host_usable"},
+                        {"name": "memory_usage_percentage",
+                         "statkey": "mem|usage_average"},
+                        {"name": "memory_balloon_kilobytes",
+                         "statkey": "mem|vmmemctl_average"},
+                        {"name": "memory_compressed_kilobytes",
+                         "statkey": "mem|compressed_average"},
+                        {"name": "memory_guest_active_kilobytes",
+                         "statkey": "mem|active_average"},
+                        {"name": "memory_consumed_kilobytes",
+                         "statkey": "mem|consumed_average"},
+                        {"name": "memory_capacity_available_to_vms_kilobytes",
+                         "statkey": "mem|totalCapacity_average"},
+                        {"name": "memory_total_capacity_kilobytes",
+                         "statkey": "mem|host_provisioned"},
+                        {"name": "summary_number_of_running_vms_kilobytes",
+                         "statkey": "summary|number_running_vms"},
+                        {"name": "network_packets_dropped_rx_number",
+                         "statkey": "net|droppedRx_summation"},
+                        {"name": "network_packets_dropped_tx_number",
+                         "statkey": "net|droppedTx_summation"},
+                        {"name": "network_packets_dropped_percentage",
+                         "statkey": "net|droppedPct"}]
+
+    def collect(self):
+        metric_list = []
+        g = GaugeMetricFamily('HostSystemCollector', os.environ["TARGET"],
```

Comment:
The naming scheme should something like `vrops_hostsystem_gauge`. 
If someone is searching for vrops metrics, he will start typing `vrops` and Prometheus will suggest all metrics from the vrops-exporter.

We **could** consider if we want to split the metric further. I.e.: `vrops_hostsystem_cpu_cores_total`, `vrops_hostsystem_cpu_sockets_info` and so on. While this is technically not necessary, it would simplify the searching process. @viennaa your opinion?

Reply:
@christopherhans i'm for splitting these metrics down as deep as needed. if we are in need of CPUs and sockes amounts then the metrics should be like you suggested. 

I also would not name the metric `_gauge`. For instrumenting them in alerts etc that information is not needed. To me it is also not clear, what this metric would mean, can you enlighten me?

Reply:
`_gauge` would only be used in the first option, because it's the type of the metric and this metric would have multiple different metrics. But you are right, wouldn't be needed for an alert. 
Option 2 would drill this down anyway..